src/main/java/frc/robot/SubSystem/HatchHolder.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.SubSystem;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class HatchHolder extends Subsystem {
+  Solenoid frontSolenoid, rightBackSolenoid, leftBackSolenoid;
+  public HatchHolder(Solenoid frontSolenoid, Solenoid rightBackSolenoid, Solenoid leftBackSolenoid){
```

Comment:
change class to DoubleSolenoid (not Solenoid) and all the functions in this class acording to this calss...
names: catchSolenoid, rightPushSolenoid, leftPushSolenoid